src/redux/books/books.js

```diff
@@ -1,28 +1,65 @@
+import axios from 'axios';
```

Comment:
- [ ] [OPTIONAL] You are not using `redux-tunk`, kindly add it to the `ConfigureStore.js` so it will be used as a middleware in the app,
e.g.
```
 import { createStore, applyMiddleware } from 'redux';
    import thunk from 'redux-thunk';
    import rootReducer from './reducers/index';

    const store = createStore(
        rootReducer,
        applyMiddleware(thunk)
    );
```

Reply:
Thank you